docs/sphinx/_ext/sphinx-mdx-builder/sphinxcontrib/mdxbuilder/writers/mdx.py

```diff
@@ -0,0 +1,806 @@
+import logging
+import re
+import textwrap
+from itertools import groupby
+from typing import TYPE_CHECKING, Any, Sequence
+
+from docutils import nodes, writers
+from docutils.nodes import Element
+from docutils.utils import column_width
+
+from sphinx import addnodes
+from sphinx.locale import admonitionlabels
+from sphinx.util.docutils import SphinxTranslator
+
+if TYPE_CHECKING:
+    from ..builders.mdx import MdxBuilder
+
+logger = logging.getLogger(__name__)
+STDINDENT = 4
+MAXWIDTH = 120
```

Comment:
Would there be benefit in making this configurable?

I see the use of the custom wrapper in `end_state()` / `do_format()`, but could you provide an example of when this would occur? For example, when we `depart_Title`, if the title was >120 characters, would it go onto a new line?